sasl_bind.c

```diff
@@ -0,0 +1,21 @@
+#include <ldap.h>
+#include "sasl_bind.h"
+
+static int
+sasl_trivial_interact (LDAP * ld, unsigned flags, void *defaults,
+                       void *sasl_interact)
+{
+  (void) ld;
+  (void) flags;
+  (void) defaults;
+  (void) sasl_interact;
+  return LDAP_SUCCESS;
+}
+
+int
+trivial_external_sasl_bind (LDAP * ld)
+{
+  return ldap_sasl_interactive_bind_s (ld, NULL, "EXTERNAL", NULL, NULL,
```

Comment:
OK, it makes sense, even though I still can't puzzle all of this together. Do you know how the authorization identity could be passed, for example? It should be an _initial client response_, but I failed to find its place after a quick scan of the API docs. If we provided at least a placeholder `String` parameter for it, the "triviality" could be documented as the lack of control support, because no other interaction can happen with the EXTERNAL mechanism anyway.